### DISCUSSION TITLE
Show video controls by default when paused

### DISCUSSION THREAD

**Discussion by mmckinley8** [TheWidlarzGroup/react-native-video#4961] [Ideas]
### What problem are you trying to solve?

I have a video that I want to start paused. This works, however, the user must tap on the paused thumbnail in order to see the controls.

### Proposed solution

The controls should show by default when the video is paused, not require the user to tap on the still image.

**Comment by marcel-happyfloat** [comment]
Yeah, somehow this behaviour changed with iOS 16. Any solution?

**Comment by nikkizol** [comment]
is there any solution?

**Comment by freeboub** [comment]
tagged as good first issue, should not be hard to implement

**Comment by marcel-happyfloat** [comment]
@vemarav any chance to have a look at this one too 🥇 ?

**Comment by Qurat-ul-ainn** [comment]
Do you fine any solution?

**Comment by adity1729** [comment]
any solution ?

**Comment by Duya3fithou** [comment]
any solution ?

**Comment by TWG-AI** [comment]
Thanks for raising this — the team will consider it. Flagged as a potential good first issue for contributors interested in picking it up.

> I'm an AI assistant — if you're interested in contributing, check the CONTRIBUTING guide for setup steps.

**Comment by Qurat-ul-ainn** [comment]
great
